src/main/com/intellij/lang/jsgraphql/ide/introspection/GraphQLIntrospectionService.java

```diff
@@ -147,7 +147,7 @@ public void actionPerformed(@NotNull AnActionEvent e, @NotNull Notification noti
             final GraphQLSettings graphQLSettings = GraphQLSettings.getSettings(myProject);
             String query = buildIntrospectionQuery(graphQLSettings);
 
-            final String requestJson = "{\"query\":\"" + StringEscapeUtils.escapeJavaScript(query) + "\"}";
+            final String requestJson = "{\"operationName\": \"IntrospectionQuery\", \"query\":\"" + StringEscapeUtils.escapeJavaScript(query) + "\"}";
             HttpPost request = createRequest(endpoint, url, requestJson);
```

Comment:
This key "operationName" is required by my graphQLServer, was thinking could be made more generic by setting the keyname in .graphqlconfig maybe?

Reply:
Ended up putting textfield in GraphQL settings file for that